str/data_frame_creator.py

```diff
@@ -0,0 +1,239 @@
+                        g_ins = attributes[9]
+                        g_stderr = attributes[9]
+                        g_qexp = attributes[9]
+                        continue
+                    g_locus_characteristics = attributes[9].split(':')
+                    g_gt = g_locus_characteristics[0]
+                    g_dp = g_locus_characteristics[1]
+                    g_q = g_locus_characteristics[2]
+                    g_repcn = g_locus_characteristics[3]
+                    g_repci = g_locus_characteristics[4]
+                    g_rc = g_locus_characteristics[5]
+                    g_enclreads = g_locus_characteristics[6]
+                    g_flnkreads = g_locus_characteristics[7]
+                    g_ml = g_locus_characteristics[8]
+                    g_ins = g_locus_characteristics[9]
+                    g_stderr = g_locus_characteristics[10]
+                    g_qexp = g_locus_characteristics[11]
+                    g_allele_1 = g_repcn.split(',')[0]
+                    g_allele_2 = g_repcn.split(',')[1]
+                    csv = csv + (
+                        '\t'.join(
+                            [
+                                sample_id,
+                                chr,
+                                start,
+                                g_gt,
+                                g_dp,
+                                g_q,
+                                g_repci,
+                                g_rc,
+                                g_enclreads,
+                                g_flnkreads,
+                                g_ml,
+                                g_ins,
+                                g_stderr,
+                                g_allele_1,
+                                g_allele_2,
+                                g_qexp,
+                            ]
+                        )
+                        + '\n'
+                    )
+    return csv
+
+
+@click.command()
+@click.option('--input-dir-eh', help='Input directory for ExpansionHunter VCFs')
+@click.option('--input-dir-gangstr', help='Input directory for GangSTR VCFs')
+@click.option('--output-name-eh', help='Output file name for ExpansionHunter eg eh.csv')
+@click.option(
+    '--output-name-gangstr', help='Output file name for GangSTR eg gangstr.tsv'
+)
+def main(input_dir_eh, input_dir_gangstr, output_name_eh, output_name_gangstr):
+    # pylint: disable=missing-function-docstring
+    # Initializing Batch
+    backend = hb.ServiceBackend(
+        billing_project=get_config()['hail']['billing_project'],
+        remote_tmpdir=remote_tmpdir(),
+    )
+    b = hb.Batch(
+        backend=backend, default_python_image=config['workflow']['driver_image']
+    )
+    j = b.new_python_job(name='EH dataframe writer')
+    g = b.new_python_job(name='GangSTR dataframe writer')
+
+    eh_csv = j.call(eh_csv_writer, input_dir_eh)
+    gangstr_csv = g.call(gangstr_csv_writer, input_dir_gangstr)
```

Comment:
As above - is there reason for one to be a TSV and one a CSV? If so, the method names should indicate which is which

Reply:
Haha, I didn't even realise this